samples/FakeItEasy.Examples.ruleset

```diff
@@ -15,4 +14,4 @@
     <Rule Id="CA2201" Action="None" />
     <Rule Id="CA2210" Action="None" />
   </Rules>
-</RuleSet>
\ No newline at end of file
+</RuleSet>

```

Comment:
Indeed. A treasured Day Job colleague and I have had this tussle in the past. Some day he'll see the light.